lesson7/dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>React Lessons</title>
+</head>
+<body>
+    <div id="app"></div>
+    <script src="bundle.js"></script>
```

Comment:
```suggestion
    <script src="/bundle.js"></script>
```
чтобы `bundle.js` запрашивался всегда из корневой папки (`dist`)